GoMeet/src/main/webapp/js/createMeetingStep2.js

```diff
@@ -0,0 +1,41 @@
+const MAX_HOURS = 23; // maximum number of hours in hour input of meeting duration
+const MAX_MINS = 59; // max number of minutes in minute input of meeting duration
+
+/**
+ * Validate that the hours and minutes entered for meeting duration are 
+ * valid.
+ * The duration should at least be 1 minute, and hours is a value 0-23
+ * and minutes 0-59. Alerts the user to input valid value(s)
+ * @returns true if duration is valid, otherwise throws an Error.
+ */
+function validateDuration() {
+  let hours = document.getElementById('duration-hours').value;
+  let mins = document.getElementById('duration-mins').value;
+  if (hours === '' || mins === '') {
+    throw(new Error(BLANK_FIELDS_ALERT));
```

Comment:
I recommend discussing with your teammates about establishing an agreed-upon set of rules when functions should and should not throw(). To contract, internally at Google, we don't use exceptions, for various practical purposes.

Reply:
Okay, will do!